core/src/main/scala/org/apache/spark/SparkConf.scala

```diff
@@ -500,6 +500,15 @@ class SparkConf(loadDefaults: Boolean) extends Cloneable with Logging with Seria
     }
   }
 
+  /**
```

Comment:
we lost the check to make sure the executor resources are a multiple of task requirements, do you want to add that back?
note I added a check (https://github.com/apache/spark/blob/master/core/src/main/scala/org/apache/spark/SparkConf.scala#L609) to make sure they were large enough but not that it was an exact fit

Reply:
Will add it back later

Reply:
added back in SparkContext.checkResourcesPerTask()